ui/page/root/trade_page.go

```diff
@@ -119,65 +119,96 @@ func (pg *TradePage) layoutMobile(gtx C) D {
 	pg.dcrdexBtn.SetEnabled(true, &gtx)
 
-	pageContent := []func(gtx C) D{
-		pg.sectionTitle(values.String(values.StrExchangeIntro)),
-		pg.layoutAddMoreRowSection(pg.exchangeBtn, values.String(values.StrExchange), pg.Theme.Icons.AddExchange.Layout16dp),
-		pg.layoutAddMoreRowSection(pg.dcrdexBtn, values.String(values.StrDcrDex), pg.Theme.Icons.DcrDex.Layout16dp),
-	}
-
 	return cryptomaterial.LinearLayout{
 		Width:     cryptomaterial.MatchParent,
 		Height:    cryptomaterial.MatchParent,
 		Direction: layout.Center,
 	}.Layout2(gtx, func(gtx C) D {
 		return cryptomaterial.LinearLayout{
-			Width:  gtx.Dp(values.MarginPadding550),
+			Width:  gtx.Dp(800),
 			Height: cryptomaterial.MatchParent,
-			Margin: layout.Inset{
-				Bottom: values.MarginPadding30,
-			},
-		}.Layout2(gtx, func(gtx C) D {
-			return pg.Theme.List(pg.scrollContainer).Layout(gtx, len(pageContent), func(gtx C, i int) D {
-				return layout.Inset{
-					Right:  values.MarginPadding48,
-					Bottom: values.MarginPadding4,
-				}.Layout(gtx, pageContent[i])
-			})
-		})
+		}.Layout(gtx,
+			layout.Rigid(func(gtx C) D {
+				return layout.Flex{Axis: layout.Vertical, Alignment: layout.Middle}.Layout(gtx,
+					layout.Rigid(func(gtx C) D {
+						return layout.Inset{Bottom: values.MarginPadding0}.Layout(gtx, pg.Theme.Label(values.TextSize20, values.String(values.StrExchangeIntro)).Layout)
+					}),
+					layout.Rigid(func(gtx C) D {
+						return layout.Inset{Bottom: values.MarginPadding16}.Layout(gtx, pg.Theme.Label(values.TextSize20, values.String(values.StrExchangeIntroPt2)).Layout)
+					}),
+
+					layout.Rigid(func(gtx C) D {
+						return cryptomaterial.LinearLayout{
+							Width:       cryptomaterial.WrapContent,
+							Height:      cryptomaterial.WrapContent,
+							Orientation: layout.Vertical,
+							Direction:   layout.Center,
+							Alignment:   layout.Middle,
+							Border:      cryptomaterial.Border{Radius: cryptomaterial.Radius(14)},
+							Padding: layout.Inset{
+								Top:    values.MarginPadding75,
+								Bottom: values.MarginPadding75,
+								Left:   values.MarginPadding180,
+								Right:  values.MarginPadding180,
+							},
+							Background: pg.Theme.Color.Gray3,
+						}.Layout2(gtx, func(gtx C) D {
+							return layout.Flex{Axis: layout.Horizontal}.Layout(gtx,
+								layout.Rigid(func(gtx C) D {
+									return pg.exchangeBtn.Layout(gtx, func(gtx C) D {
+										return cryptomaterial.LinearLayout{
+											Width:       160,
+											Height:      180,
+											Orientation: layout.Vertical,
+											Direction:   layout.Center,
+											Alignment:   layout.Middle,
+											Padding:     layout.UniformInset(20),
+											Border:      cryptomaterial.Border{Radius: cryptomaterial.Radius(14)},
+											Background:  pg.Theme.Color.Surface,
+										}.Layout(gtx,
+											layout.Rigid(func(gtx C) D {
+												return pg.Theme.Icons.TradeExchange.LayoutSize(gtx, values.MarginPadding70)
```

Comment:
Why use 70? Your icons are currently overly large compared to the design.

Reply:
Here is how it looks with the suggested changes
![Screenshot 2023-10-02 101848](https://github.com/crypto-power/cryptopower/assets/54014025/f21c3096-33ef-4e09-959d-5944700bbae8)